src/ensembl/src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -57,6 +60,15 @@ export const SpeciesSearchField = (props: Props) => {
 
   const onMatchSelected = (match: SearchMatch) => {
     props.onMatchSelected(match);
+
+    const speciesName = `${match.common_name || match.scientific_name} - ${
+      match.assembly_name
+    }`;
```

Comment:
Why is the species name here set differently than in PopularSpeciesButton or in speciesSelectorActions (where it doesn't include assembly name)? Is it intentional?

Reply:
It was just species name in the document but it makes sense to include the assembly name as well. I've updated it.